src/main/kotlin/org/opensearch/replication/action/stats/LeaderStatsRequest.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.replication.action.stats
+
+import org.opensearch.action.support.nodes.BaseNodesRequest
+import org.opensearch.common.io.stream.StreamInput
+import org.opensearch.common.io.stream.StreamOutput
+import java.io.IOException
+
+/**
+ * A request to get node (cluster) level replication stats.
+ */
+class LeaderStatsRequest : BaseNodesRequest<LeaderStatsRequest?> {
+
+    /**
+     *
+     * @param in A stream input object.
+     * @throws IOException if the stream cannot be deserialized.
+     */
+    constructor(`in`: StreamInput) : super(`in`)
```

Comment:
nit: we can instead not use reserved keywords - maybe `inp`
Have to change in below file as well